app/controllers/merchant/item_orders_controller.rb

```diff
@@ -2,9 +2,11 @@ class Merchant::ItemOrdersController < Merchant::BaseController
   def update
     item_order = ItemOrder.find(params[:id])
     item = Item.find(params[:item_id])
+    order = Order.find(item_order.order_id)
       item.update_attribute(:inventory, item.inventory - item_order.quantity)
       item_order.update_attribute(:status, "fulfilled")
       flash[:success] = "You have fulfilled #{item.name}"
+      order.packaged_check
```

Comment:
This method lives in order.rb and runs anytime an item is added to an order to ensure new items get packaged.